src/quo2/components/loaders/skeleton/view.cljs

```diff
@@ -2,45 +2,68 @@
   (:require [quo2.theme :as theme]
             [react-native.core :as rn]
             [quo2.foundations.colors :as colors]
+            [react-native.masked-view :as masked-view]
+            [react-native.reanimated :as reanimated]
             [quo2.components.loaders.skeleton.style :as style]
+            [reagent.core :as reagent]
             [quo2.components.loaders.skeleton.constants :as constants]))
 
-(defn- skeleton-item
-  [index content color]
-  [rn/view {:style (style/container content)}
-   [rn/view {:style (style/avatar color)}]
-   [rn/view {:style style/content-container}
-    [rn/view
-     {:style (style/content-view
-              {:type    (if (= content :list-items) :message :author)
-               :index   index
-               :content content
-               :color   color})}]
-    [rn/view
-     {:style (style/content-view
-              {:type    (if (= content :list-items) :author :message)
-               :index   index
-               :content content
-               :color   color})}]
-    (when (= content :notifications)
-      [rn/view
-       {:style (style/content-view {:type    :message2
-                                    :index   index
-                                    :content content
-                                    :color   color})}])]])
+(defn skeleton-item
+  [index content color animated?]
```

Comment:
use map as it gives more flexibility to add or remove keys without much refactoring.

```suggestion
  [{:keys [index content color animated?]}]
```

Reply:
tks. Its done =) I also updated the video with darkmode and the changes/refactors